prestodb/client.py

```diff
@@ -199,6 +201,7 @@ def __init__(
         schema=None,  # type: Text
         session_properties=None,  # type: Optional[Dict[Text, Any]]
         http_headers=None,  # type: Optional[Dict[Text, Text]]
+        transaction_id='NONE',  # type: Optional[Text]
```

Comment:
Given we have a property interface for this, can we change the name to `_ transaction_id `

Reply:
The property implements a setter that proxies the assignment to `PrestoRequest._client_session.transaction_id`. I would use `_transaction_id` if `PrestoRequest` was managing the state this value itself.